packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -495,6 +496,12 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         VoteInstance storage voteInstance = priceRequest.voteInstances[currentRoundId];
         voteInstance.voteSubmissions[voter].commit = hash;
 
+        // There is a very specific edge case that can occur if the only participates in a vote stakes after the request
+        // is created and the request is rolled. If this happens, a request becomes unresolvable due to lastVotingRound
+        // not being correctly updated. To combat this, we can check this assignment here. This is a no-op in all
+        // cases except the edge case specified as this should be assigned correctly elsewhere.
+        if (priceRequest.lastVotingRound != currentRoundId) priceRequest.lastVotingRound = currentRoundId;
```

Comment:
nit: should we remove the round increments from the rolling action? Or do we need to keep that in place? I assume we still need it for some of the rolling/round tracking details, right?

Reply:
you need it to happen there _or_ here. you should still be able to roll and slash without calling commit. i.e if you withdraw rewards, for example, we should update your tracker and activate the rolled logic.